.circleci/config.yml

```diff
@@ -5,24 +5,24 @@ jobs:
     steps:
       - checkout
       - run: ./git-submodule-checkout.sh
-      - run: ./docker-build.sh
-      - run: ./docker-test.sh
+      - run: ./docker/build/build.sh
+      - run: ./docker/test/test.sh
```

Comment:
The `./docker` folder is not really just for docker files, it also has `test.sh` which we use regularly outside docker to test the project

I recommend we rename `./docker` to `./_build` - it's all files related to the project build (docker + test)

The underscore in the beginning is a good convention to indicate this directory doesn't contain go code (since go build ignores directories starting with underscore)

A second fix would be do rename the internal folder `./docker/build` to `./build/docker` because `./build/build` is poop :)

Reply:
what about `./_docker` then?